update.mjs

```diff
@@ -6,6 +6,8 @@ const buildTools = [
   'rollup.config.*',
   'webpack.config.*',
   'gulp*',
+  'nodemon.*',
```

Comment:
We have this already

Reply:
nodemon* is also available for configuration, for example: nodemon.json